packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Documents.Example.tsx

```diff
@@ -230,7 +229,9 @@ export class DetailsListDocumentsExample extends React.Component<any, IDetailsLi
 
   public componentDidUpdate(previousProps: any, previousState: IDetailsListDocumentsExampleState) {
     if (previousState.isModalSelection !== this.state.isModalSelection) {
-      this._selection.setModal(this.state.isModalSelection);
+      if (this.state.isModalSelection === false) {
```

Comment:
Not a big deal, but `isModalSelection` is typed as a `boolean` so _I think_ the `===` is unnecessary

https://github.com/OfficeDev/office-ui-fabric-react/blob/5bc73a304b843bf183bf77659e0165d395f4af91/packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Documents.Example.tsx#L41

It can be simplified to: `if (!this.state.isModalSelection) {`

Reply:
Good catch!